### DISCUSSION TITLE
Welcome to JSUtilityKit Discussions!

### DISCUSSION THREAD

**Discussion by Mad-Man-Dog** [Mad-Man-Dog/JSUtilityKit#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Mad-Man-Dog** [comment]
You can ask anything you want here

**Reply by ayeshanweerasuriya** [reply]
this is a testing of commenting and creating discussions

**Comment by ayeshanweerasuriya** [comment]
what is the elephant's wife name?
